pyproject.toml

```diff
@@ -10,17 +10,19 @@ packages = [
     { include = "dotnet", from = "src" },
 ]
 [tool.poetry.dependencies]
-python = ">=3.10,<4.0"
+python = "^3.11"
```

Comment:
is it intentional that we drop 3.10 here?

Reply:
yes, `py_obs` depends on `python = "^3.11"`.